frontend/src/components/MeetingRoom/TimeRoomName/TimeRoomName.tsx

```diff
@@ -13,7 +13,12 @@ const TimeRoomName = (): ReactElement => {
   const roomName = useRoomName();
 
   return (
-    <div className="ml-3 mt-1 truncate font-normal text-white">
+    <div
+      className="ml-3 mt-1 truncate font-normal text-white"
+      style={{
+        flex: '1 0 0%',
```

Comment:
what that `flex: 1 0 0%` do in this case? if possible it'd be nice not to mix `className`s with the `style`s  - I know we do it a lot but it'd be nice not to introduce anything new like that if we don't have to 🙏

Reply:
We can go back to it here. Nice catch!